### PR TITLE
fixed some typo/logging issues

### PR DESCRIPTION
**Pull request by bsmithvt**

### CODE REVIEW

File: Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1 [hunk 1]
```diff
@@ -217,7 +217,6 @@ function Move-ActiveDirectoryGroupsToAzureAD {
                     Write-DosMessage -Level Information -Message "Found group $($azureADGroup.DisplayName) with ObjectId $($azureADGroup.ObjectId) in Azure AD Tenant $($tenantId)"
                 }
                 else {
```
Comment (ctediz) [inline Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1]: this else can probably removed right?

File: Fabric.Authorization.API/scripts/AzureAD-Utilities.psm1 [hunk 2]
```diff
@@ -24,13 +24,13 @@ if ($azureAD.length -eq 0) {
     $installed = Get-InstalledModule -Name AzureAD -ErrorAction "silentlycontinue"
 
     if (($null -eq $installed) -or ($installed.Version.CompareTo($minVersion) -lt 0)) {
-        Write-Host "Instgalling AzureAD from Powershell Gallery"
+        Write-DosMessage "Installing AzureAD from Powershell Gallery"
```
Comment (michaelvidal24) [inline Fabric.Authorization.API/scripts/AzureAD-Utilities.psm1]: Does Write-DosMessage require the -Level parameter?

File: Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1 (hunk 1 above)
Comment (bsmithvt) [reply Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1]: It prevents the second `if` check below. So I suppose we could remove that check or this `else`.

Comment (ctediz) [reply Fabric.Authorization.API/scripts/AzureAD-Groups-Migration.ps1]: Ah, I assumed the catch was the end of the foreach

**Pull request merged**